examples/amp-story/amp-story-captions.html

```diff
@@ -74,6 +79,42 @@
       </amp-story-grid-layer>
     </amp-story-page>
 
+    <amp-story-page id="page-captions-preset-default" auto-advance-after="video-container">
+      <amp-story-grid-layer template="fill">
+        <amp-video id="video-container"
+            src="https://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ElephantsDream.mp4"
+            width="1280" height="720" layout="responsive" autoplay captions-id="captions-preset-default">
+          <track kind="subtitles" src="captions-for-container-layout.vtt" srclang="en" default>
+        </amp-video>
+      </amp-story-grid-layer>
+      <amp-story-grid-layer template="vertical" class="bottom">
+        <div class="scrim">
+          <div class="static-text">Default preset example.</div>
+          <div class="static-text">This element is always above captions and never overlaps.</div>
+          <amp-story-captions data-preset="default" id="captions-preset-default" layout="container"></amp-story-captions>
+          <div class="static-text">This element is always below captions and never overlaps.</div>
+        </div>
+      </amp-story-grid-layer>
+    </amp-story-page>
+
+    <amp-story-page id="page-captions-preset-appear" auto-advance-after="video-container">
+      <amp-story-grid-layer template="fill">
+        <amp-video id="video-container"
+            src="https://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ElephantsDream.mp4"
+            width="1280" height="720" layout="responsive" autoplay captions-id="captions-preset-appear">
+          <track kind="subtitles" src="captions-for-container-layout.vtt" srclang="en" default>
+        </amp-video>
+      </amp-story-grid-layer>
+      <amp-story-grid-layer template="vertical" class="bottom">
+        <div class="scrim">
+          <div class="static-text">Appear preset example.</div>
+          <div class="static-text">This element is always above captions and never overlaps.</div>
+          <amp-story-captions data-preset="appear" style="--amp-story-captions-font-size: .8em; --amp-story-captions-font-family: 'Shadows Into Light'" id="captions-preset-appear" layout="container"></amp-story-captions>
```

Comment:
As an official API to the custom element, do we need to use `data-` prefixing the element? All other components use regular non-data attributes for configuration purposes, and validating `data-` attribute sounds counterintuitive to me

Reply:
We do both in the codebase, I'd have went with no `data-` prefix as well, but it doesn't really matter. Both work. The only advantage of no `data-` prefix is that it better protects against typos.
`data-perset=` would be valid, while `perset=` would not.

Reply:
Updated this to `style-preset`.